griptape/observability/observability.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional
+from attrs import define, field
+from griptape.drivers.observability.base_observability_driver import BaseObservabilityDriver
+from griptape.drivers.observability.dummy_observability_driver import DummyObservabilityDriver
+
+_dummy_observability_driver = DummyObservabilityDriver()
+_global_observability_driver: Optional[BaseObservabilityDriver] = None
+
+
+@define
+class Observability:
+    driver: BaseObservabilityDriver = field(kw_only=True)
```

Comment:
Rename `driver` to `observability_driver` for consistency with other uses of Drivers.

Reply:
Renamed!